app/translations/messages.de_DE.json

```diff
@@ -0,0 +1,4 @@
+{
+  "Your inquiry at Wikimdia": "Ihre Anfrage an Wikimedia",
```

Comment:
1. I agree that when the real translation feature is implemented, we should exchange the english strings in the code against translation keys.
2. Just for consistency. The translator needs to be initialized with a full locale string, so I was using the same locale for the message file.

The missing e is consistent in both code and message file, but I'll correct it.